zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -555,7 +566,11 @@ angular.module('zeppelinWebApp')
     if (isNaN(timeMs) || timeMs < 0) {
       return '&nbsp;';
     }
-    return 'Took ' + (timeMs/1000) + ' seconds';
+    var desc = 'Took ' + (timeMs/1000) + ' seconds.';
+    if (pdata.textSaved !==undefined && Date.parse(pdata.textSaved) > Date.parse(pdata.dateStarted)){
+      desc += ' (outdated)'
+    }
+    return desc;
```

Comment:
@felixcheung  thanks for your feedback. 
I will fix 'outdated' logic and  use `ng-disabled` instead of `ng-show` for consistency